### ISSUE TITLE
Missing "Wow effect"

### ISSUE THREAD

**Issue by ciscokid009** [Haoming02/sd-webui-forge-classic#776]
By far not high priority, (still counting/sorting all the mind-blowing features/possibilities you wizzarded recently), one little feature of good old webui got under the bus: **Png-Info**

`It still shifts the images, but gen data + prompts remain untouched.`

But this gem now is valuable like never before, I'd imagine.
Feeding new tech with old intensions of older generations - the missing "Wow effect" will likely be followed by kinda "Holy sh*t effect" when driving old intesions further up the road.

Shifting cfg and batch size wouldn't make much sense because of new architecture's demandings, but prompts and dim still do.

Update: figured out that sometimes it works, sometimes not, but I can't really explain why, preset related somehow?
Maybe a kind of "compatibility ckeckbox" is a cheap "fit for all" way, just shift the minimalzed basics every arch can deal with.

**Comment by Haoming02** [comment]
Urmmm...

What?

**Comment by ciscokid009** [comment]
: )
concrete use case of mine: 
creating graphic novels, highly styled, art scope

sources i've made with sdxl/pony for locations, poses, objects,...
styles/characters/locations, all trained with lora only, finetune has been impossible with my weak rig.

results are tons of scenes with more or less unsatisfying anatomy, inconsistent and varying characters, locations, styles.

especially in creating graphic novels it's required that the hero's anatomy/outfits or recurring cars, locations, his home for example always looks the same, this was nearly impossible  with old tech without high efforts in photoshop post processing

new architecture can do this, even without lora.

I dropped these old sdxl based generated images into png-info, but clicking "send to txt2image/img2img" transfered the image, but not the width/height of the image, most fatal - not the promp's to img2img, although all generation info was displayed in png-info.

as a workaround i did this manually and regenerated with flux klein. the outcome, only using old prompts, was amazing.

i would prefer a simple"send to"- button click rather than copy promps manually from png-info tab to txt2img/img2img tabs.

i don't know what's causing this (at least on my install), but i noticed the metadata stored in image's comment now reflects new tech.
probably this is a spot where a problem could exist.

will check further.

**Comment by abzaloff** [comment]
Hmm, I just tried to reproduce your issue and found that there are no problems. When clicking the “Send to txt2img” button, the image is sent with all its parameters, and the dimensions are applied as well.

You can also drag the image not into PNG info, but directly into the prompt field, and then click that arrow. All parameters from the image will be applied.

<img width="136" height="73" alt="Image" src="https://github.com/user-attachments/assets/c337b3c8-acee-4fdc-9493-54aa507b808f" />

**Comment by Haoming02** [comment]
Urmmm...

What? (2)

https://github.com/user-attachments/assets/90ca003c-e040-41a4-b231-de7a35ae8446

**Comment by ciscokid009** [comment]
@abzaloff 
"You can also drag the image not into PNG info, but directly into the prompt field, and then click that arrow. All parameters from the image will be applied."
Wow effect! :D 
Didn't know of this possibility! Thanks for that, very much appreciated!

I too can't reproduce this issue anymore, don't remember the images (10-12, various creation dates in past). I use a little python script that creates captions text files from generations data embeded in images,  i will try to modify it to run it over my image repositories too in paralle for searching for the images that showed correct info, but the transfer didn't work out. 

As for now it looks like the issue is only my side.
